@JonLiu1993  There are not really errors. Meson performs tests to check if certain functions exist. I get the same stuff
```
Compiler stderr:
 /home/theartful/Programming/vcpkg/buildtrees/glib/x64-linux-dbg/meson-private/tmp68g4jd6k/testfile.c: In function 'main':
/home/theartful/Programming/vcpkg/buildtrees/glib/x64-linux-dbg/meson-private/tmp68g4jd6k/testfile.c:17:18: error: #error "__builtin_getvfsstat not found"
   17 |                 #error "__builtin_getvfsstat not found"
      |                  ^~~~~

Checking for function "getvfsstat" : NO 
```
The cause of the error is something else. Can you please send me the log files?

--- a/ports/glib/portfile.cmake
+++ b/ports/glib/portfile.cmake
@@ -17,7 +17,7 @@ vcpkg_extract_source_archive(
 
 if (selinux IN_LIST FEATURES)
     if(NOT VCPKG_TARGET_IS_WINDOWS AND NOT EXISTS "/usr/include/selinux")
-        message("Selinux was not found in its typical system location. Your build may fail. You can install Selinux with \"apt-get install selinux\".")
+        message("Selinux was not found in its typical system location. Your build may fail. You can install Selinux with \"apt-get install selinux libselinux1-dev\".")
     endif()
     list(APPEND OPTIONS -Dselinux=enabled)
 else()

--- a/versions/g-/glib.json
+++ b/versions/g-/glib.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "52b87ac2b4be057c665b94927a552e131e5a920c",
+      "git-tree": "97fc41e084d04073610421cbdf4bfae0e89fb99c",
       "version": "2.74.0",
       "port-version": 0
     },

